packages/story-editor/src/components/form/textArea.js

```diff
@@ -38,7 +38,7 @@ const TextArea = forwardRef(
       placeholder,
       value,
       disabled,
-      rows,
+      rows = 2,
```

Comment:
Looks like this prop is never used, yet in some places (`ImageAccessibilityPanel`, `VideoAccessibilityPanel` and `ExcerptPanel`) we pass it. But it doesn't actually have an effect.

Reply:
Should I remove it?

Reply:
Once confirmed it has no effect, yes sure

Reply:
It does seem to be used in `ExcerptPanel`. Fixed now.